service/src/main/resources/db/changesets/20221012_initial.yaml

```diff
@@ -20,7 +20,13 @@ databaseChangeLog:
                     nullable: false
                   defaultValue: "{}"
                   remarks: Schemaless object storing the profile data
-        -  sqlFile:
-            splitStatements:  false
+        - sqlFile:
+            splitStatements: false
             relativeToChangelogFile: true
-            path:  functions/pathfunc.sql
+            path: functions/pathfunc.sql
+        - sql:
+            # 32MB Upper bound per (potentially compressed) profile object.
+            #
+            # liquibase has checkConstraint in its column definition, but it
+            # doesn't seem to do anything.
+            sql: ALTER TABLE user_profile ADD CONSTRAINT max_profile_size CHECK (pg_column_size(profile_obj) < 33554432);
```

Comment:
We usually make a new changeset, rather than modify an existing one. Not sure if this will work (maybe it will).

Reply:
+1. liquibase shouldn't allow making changes to already published file. I'm surprised you didn't get an error about this at run time. might be worth checking if liquibase is set up correctly...

Reply:
Right, I thought I could merge the changesets since I remember hearing liquibase merging was slow, but forgot to consider that this had already been deployed!